packages/astro/src/core/config.ts

```diff
@@ -397,7 +397,59 @@ export async function resolveConfigURL(
 	}
 }
 
-/** Attempt to load an `astro.config.mjs` file */
+interface OpenConfigResult {
+	userConfig: AstroUserConfig;
+	astroConfig: AstroConfig;
+	flags: CLIFlags;
+	root: string;
+}
+
+/** Load a configuration file, returning both the userConfig and astroConfig */
+export async function openConfig(configOptions: LoadConfigOptions): Promise<OpenConfigResult> {
```

Comment:
Assuming you're renaming due to the changed API signature?

Reply:
Yep. Left the other in as deprecated.